setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude=invoke/vendor,sites,tests
+ignore=E124,E125,E128,E261,E302,E402,F401,W503,E712,F821
```

Comment:
Yeah, I figure start with a lot of them ignored. If individual ones get fixed they can be removed from this list.